src/main/resources/com/cloudbees/plugins/credentials/CredentialsStoreAction/CredentialsWrapper/index.properties

```
@@ -22,3 +22,5 @@
 # THE SOFTWARE.
 #
 usageTrackingImperfect=Note: usage tracking requires the cooperation of plugins and consequently may not track every use.
+fingerprintDisabledWarning=Credential fingerprints are currently disabled which is required to track their usage. \
```

There are a couple of problems with this "which" clause.
I suggest something simple like this.
```suggestion
fingerprintDisabledWarning=Credential fingerprints are disabled. \
```